app/validators/metadata_validator.py

```diff
@@ -0,0 +1,40 @@
+from functools import cached_property
+
+from app.validators.questionnaire_schema import find_duplicates
+from app.validators.validator import Validator
+
+
+class MetadataValidator(Validator):
+    FOUND_MISSING_METADATA = "Metadata not specified in metadata field"
+    FOUND_DUPLICATE_METADATA = "Metadata contains duplicates"
```

Comment:
Same here, don't think we need `FOUND_`

Reply:
I'm going to disagree on these - they're framed as events to hint at the fact they're errors rather than static vars.